internal/guestrequest/types.go

```diff
@@ -46,7 +46,7 @@ type LCOWMappedDirectory struct {
 // Read-only layers over VPMem
 type LCOWMappedVPMemDevice struct {
 	DeviceNumber uint32 `json:"DeviceNumber,omitempty"`
-	MountPath    string `json:"MountPath,omitempty"` // /tmp/pN
+	MountPath    string `json:"MountPath,omitempty"` // /run/layers/pN
```

Comment:
Lets just remove this comment since it can be anywhere. The documentation should be on the caller not the struct.

Reply:
Same for `LCOWMappedVirtualDisk.MountPath`